### PR TITLE
New updates : typos  fixed + Word Cloud more complete + Dependencies' versions updated

### PR DESCRIPTION
**Pull request by younes-io**

### CODE REVIEW

**Comment by aybbou** [comment]
Again the test is failing and the test files are deleted ! this is weird !? Did you run the tests locally ?

**Comment by younes-io** [comment]
Don't you see that I didn't delete any tests ? 
I have created another fork that I cloned, then I made the changes on the view (take a look at the commit https://github.com/younesherlock/Rayak/commit/bc3bdda20ebcef1eb69067528c7c82cc5fa946e6)
It's working locally.
The build on Travis fails because the tests should be adapted to the new updates (The number of keywords is divided by 1.3 for a better rendering of the Word Cloud)

**Comment by aybbou** [comment]
By "working locally" I mean that the PHPUnit tests are passing. If not your should rewrite the tests so they pass.
And yeah the files are not deleted. I am just tired :tired_face:

File: src/Db/MainBundle/Resources/views/Default/index.html.twig [hunk 1]
```diff
@@ -35,12 +35,6 @@
 <div class="navbar navbar-default navbar-fixed-top" role="navigation">
     <div class="container">
         <div class="navbar-header">
-            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
-                <span class="sr-only">Toggle navigation</span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-                <span class="icon-bar"></span>
-            </button>
```
Comment (aybbou) [inline src/Db/MainBundle/Resources/views/Default/index.html.twig:43]: Isn't this for the responsive mobile menu ?

**Comment by younes-io** [comment]
Alright. I'll adapt the tests as soon as I can =)

**Pull request closed without merging**